ruby/blue_steel/Rakefile

```diff
@@ -0,0 +1,6 @@
+require 'bundler/gem_tasks'
+load 'lib/tasks/style.rake'
+task default: %i[style:branch]
+
+task :environment do
```

Comment:
This doesn't seem to do anything. You probably want to remove it or am I missing something?